manifest.json

```diff
@@ -400,6 +400,30 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/ubuntu/18.04/cross/freebsd/11",
+              "os": "linux",
+              "osVersion": "freebsd.11",
```

Comment:
```suggestion
              "osVersion": "ubuntu18.04",
```

Reply:
updated. Is there place in manifest to reflect target OS/arch? Or is it only by the tag?